ynetetskaya/01_HTML/test_03.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+  <HTML>
+    <HEAD>
+      <META charset="utf-8">
+      <TITLE>Python 02</TITLE>
+    </HEAD>
+    <BODY>
+        <H1>HOW-TO: Программа на Python, Часть 2</H1>
+        <UL>
+            <LI><EM>Программа на Python — часть 1</EM></LI>
+            <LI><EM>Программа на Python — часть 2</EM></LI>
+            <LI><EM>Программа на Python — часть 3</EM></LI>            
+        </UL>
+        <H2>Содержание</H2>
+        <H3>HOW-TO: Программа на Python, Часть 2</H3>
+        <UL>
+            <LI><INS>Списки</INS></LI>
+            <LI><INS>Комментарии</INS></LI>
+        </UL>
+        <HR>
+        <P>Впрошлом уроке мы рассмотрели простую программу, которая использовала raw_input
+        для получения ответа пользователя, простые типы переменных и простой цикл «<STRONG>for</STRONG>». В
+        этом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше
+        программ.</P>
+        <H2>Списки</H2>
+        <P>Давайте рассмотрим другой тип переменных, который называется <EM>списки</EM>. В других
+        языках программирования они называются <EM>массивами</EM>. Если мы обратимся к аналогии с
+        коробками для обуви, то массив (список) представляет собой несколько склеенных друг с
+        другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в
+        другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы
+        запрограммируем его так:</P>
+        <P></P>
```

Comment:
Could you please add extra line inside pre section below, then using empty paragraph here